.github/workflows/ci.yml

```diff
@@ -67,6 +67,11 @@ jobs:
             ninja-build \
             pkg-config
 
+      - name: Install uncrustify
+        run: |
+          brew update --quiet
+          brew install uncrustify
```

Comment:
I guess the `apt-get` package (if there is one) is too old?

Reply:
Yeah.